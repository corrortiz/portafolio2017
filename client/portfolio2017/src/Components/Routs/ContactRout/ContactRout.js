import React from 'react';

const ContactRout = () => {
  return (
    <div>
      <h1>Contact Rout, Beacthhh</h1>
    </div>
  );
};

export default ContactRout;
