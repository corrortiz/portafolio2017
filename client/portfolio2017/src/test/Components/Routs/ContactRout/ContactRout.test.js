// @flow
import React from 'react';
import { shallow } from 'enzyme';
import ContactRout from '../../../../Components/Routs/ContactRout/ContactRout';

it('should compare a snapshot of ContactRout', () => {
  const wrapper = shallow(<ContactRout />);
  expect(wrapper).toMatchSnapshot();
});
