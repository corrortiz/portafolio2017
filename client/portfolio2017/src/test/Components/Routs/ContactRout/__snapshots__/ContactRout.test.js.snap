// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should compare a snapshot of ContactRout 1`] = `
<div>
  <h1>
    Contact Rout, Beacthhh
  </h1>
</div>
`;
